package/MDAnalysis/topology/MOL2Parser.py

```diff
@@ -93,6 +97,13 @@ def parse(self, **kwargs):
         Returns
         -------
         A MDAnalysis Topology object
+
+
+        .. versionchanges: 2.0.0
+           Parse elements from atom types. The elements attribute can contain
+           MOL2-specific atom types such as Du (dummy atom), Any,
+           Hal (halogen), Het (heteroatom) and Hev (heavy atoms).
```

Comment:
What do you think about this? Given that the actual types are stored in the `types` attribute, I think the `elements` attribute should only contain actual elements (that can be readily converted to atomic numbers, for example). In such case, `Du.C` should be treated as carbon?

Reply:
Yeah I don't think I agree with placing `Du` in elements, I think our general approach is that dummy atoms are empty strings. If users want to use the types they can directly access the `atomtypes` attribute.

Regarding `Du.C`, I would consider that a dummy atom, therefore no element. Maybe @lilyminium has a different view here? I generally think that if folks have gone through the trouble of making something a dummy atom, then it should be treated as such.

"Any, Hal (halogen), Het (heteroatom) and Hev (heavy atoms)" - I'm not sure what you mean here, could you explain this?